src/nectarchain/dqm/pixel_participation.py

```diff
@@ -0,0 +1,148 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelParticipation_HighLowGain(DQMSummary):
+    def __init__(self, gaink):
+        self.k = gaink
+        return None
```

Comment:
`__init__` should not return anything.

Reply:
this also applies for the entire dqm as there is a return None in every __init__

Reply:
done